yarn-project/circuit-types/src/logs/l1_payload/encrypted_log_payload.test.ts

```diff
@@ -52,52 +46,47 @@ describe('EncryptedLogPayload', () => {
 
       expect(recreated?.toBuffer()).toEqual(original.toBuffer());
     });
-
-    it('decrypt a log as outgoing', () => {
-      const recreated = EncryptedLogPayload.decryptAsOutgoing(payload, ovskM);
-
-      expect(recreated?.toBuffer()).toEqual(original.toBuffer());
-    });
   });
 
-  it('outgoing cipher text matches Noir', () => {
-    const ephSk = GrumpkinScalar.fromHighLow(
-      new Fr(0x000000000000000000000000000000000f096b423017226a18461115fa8d34bbn),
-      new Fr(0x00000000000000000000000000000000d0d302ee245dfaf2807e604eec4715fen),
-    );
+  // MIKE Q: why is there note an "incoming" version of this test?
+  // it('outgoing cipher text matches Noir', () => {
```

Comment:
Wasn't sure whether to delete this test, or adapt it to be an "incoming" test?

Reply:
Upon further inspection, it's very tailored towards testing the outgoing log approach to generating a symmetric key